content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_diff.md

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Please wait for the doc generator issue to be fixed. This 1.11 branch won't get published any soon so it is not an urgent problem. If you have believe there are issues to be fixed in the doc content, please raise it to kubeadm source repo.

Reply:
we have issues for everything in the `kubernetes/kubeadm` repo. i was told by the kubeadm maintainers to start submitting PRs for these new `kubeadm` commands which implies including them (`< include ...`).

why isn't the 1.11 branch of `website` going to be merged at release?